setup/copy_course_from_production.rb

```diff
@@ -50,3 +51,8 @@ def make_copy_of(url)
   pp "http://localhost:3000/courses/#{course.slug}"
 end
 
+# When parsing update_logs from flags, keys are set as strings instead of integers
+# This causes problems, so we need to force the keys to be integers.
+def fix_update_logs_parsing(update_logs)
+  update_logs.transform_keys(&:to_i)
+end
```

Comment:
There is a lot of code duplication, do we want to keep this command?

Reply:
No, this file should be removed.

Reply:
Done on fff45d60a71334b38b007bfaab1c18b73c09c91d